recipes/valet/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+mkdir -p $PREFIX/bin
+
+cp src/py/* $PREFIX/bin
+export VALET=$PREFIX/bin
```

Comment:
Is this export needed at runtime?

Reply:
yes. Is there a different way to do that?

Reply:
This is currently only valid in the build environment.
If you would have a single binary, I would recommend a wrapper. E.g. a file called `_binary` that is a shell script that `export VLATE=`which binary` && binary`.
In your case it is probably better to patch the python scripts to automatically set VALET to `os.path.realpath(__file__)`

Reply:
:) 
I was testing the second option